stl/inc/ranges

```diff
@@ -29,6 +29,69 @@ namespace ranges {
     template <class _Rng>
     concept viewable_range = range<_Rng>
         && (borrowed_range<_Rng> || view<remove_cvref_t<_Rng>>);
+
+
+    // CLASS ranges::ref_view
+    // clang-format off
+    template <range _Rng>
+        requires is_object_v<_Rng>
+    class ref_view : public view_interface<ref_view<_Rng>> {
+        // clang-format on
+        void _Rvalue_poison(_Rng&);
+        void _Rvalue_poison(_Rng&&) = delete;
+
+    public:
+        constexpr ref_view() noexcept = default;
+
+        // clang-format off
+        template <_Not_same_as<ref_view> _OtherRng>
+        //     requires convertible_to<_OtherRng, _Rng&> && requires { _Rvalue_poison(declval<_OtherRng>()); }
+        constexpr ref_view(_OtherRng&& _Range)

```

Comment:
Nitpick: I think it's confusing to have a `_Rng` template parameter belonging to the class, followed by a `_Range` function parameter that is potentially a different type. I would prefer to see the function parameter be named `_Other` or really anything else.

(Even with concepts, in template code, naming is one of our few defenses against confusion and horrible compiler errors.)

Reply:
Changed to `_Other`